src/plugins/ui_feature/data_cleanup.ts

```diff
@@ -202,4 +206,43 @@ export class DataCleanupPlugin extends UIPlugin {
     }
     return CommandResult.Success;
   }
+
+  private trimWhitespace() {
+    const zones = this.getters.getSelectedZones();
+    const sheetId = this.getters.getActiveSheetId();
+    let count = 0;
+    for (const zone of zones) {
+      positions(zone).map(({ col, row }) => {
```

Comment:
don't use `map` if you don't care about the returned value. It's building an array uselessly
```suggestion
    for (const { col, row } of zones.map(positions).flat()) {
```